test/osutils.jl

```diff
@@ -0,0 +1,8 @@
+@test (@static true ? 1 : 2) == 1
```

Comment:
how about test/misc.jl. there are only a couple of uses of at-static in existing tests, it doesn't appear to have dedicated unit tests of its own yet